config/locales/en.yml

```diff
@@ -200,6 +200,9 @@ en:
   email_address_added: Email address added
   email_address_removed: Email address removed
 
+# controllers/private_messages
+  private_message_at: posted on -
```

Comment:
You can change it to the following to make it work correctly:
`private_message_sent_at: 'posted by %{email} on %{at}'`